storybook/components/snippet/Snippet.css

```diff
@@ -7,5 +7,10 @@
     padding: var(--o-ui-sp-5) !important;
     font-size: 15px;
     border-radius: 4px;
+    overflow-x: auto;
     margin: 0;
 }
+
+.o-ui-sb-snippet .token-line {
```

Comment:
Is it in the wrong file? I don't understand how a token line could be in a code snippet?

Reply:
> Is it in the wrong file? I don't understand how a token line could be in a code snippet?

![image](https://user-images.githubusercontent.com/361632/152035612-104fbf36-c3b9-4c43-8392-45e3f36ce7f7.png)

a token line is each line of a snippet.

Reply:
ah got it, I though it's was our tokens (tokens page)

Reply:
Nope